lib/image-diff.js

```diff
@@ -54,12 +54,13 @@ ImageDiff.createDiff = function (options, cb) {
   // http://www.imagemagick.org/script/compare.php
   var diffCmd = 'compare';
   var diffArgs = [
-    'compare',
+//    'compare',
     '-verbose',
     // TODO: metric and highlight could become constructor options
     '-metric', 'RMSE',
+    '-dissimilarity-threshold', '1',
     '-highlight-color', 'RED',
-    '-compose', 'Src',
+    options.shadow[0], options.shadow[1],
```

Comment:
Maybe we can prepare the options as an array and then concatenate `shadow` options on? Then, attach the path parameters after all the options are read.

```js
var diffOptions = [
  '-verbose',
  '-metric', 'RMSE',
  '-highlight-color', 'RED'
];
if (options.shadow) {
  diffOptions = diffOptions.concat([
    '-dissimilarity-threshold', '1'
  ]);
} else {
  diffOptions = diffOptions.concat([
    '-compose', 'Src'
  ]);
}

var diffArgs = diffOptions.concat([
  options.actualPath,
  options.expectedPath,
  // If there is no output image, then output to `stdout` (which is ignored)
  options.diffPath || '-'
]);
```